File: OnlineJudge/P12403.java
import java.util.*;

public class SaveSetu {

    public static void main(String[] args) {
        
        Scanner s = new Scanner(System.in);

        int operations = s.nextInt();
        long sum = 0;

        for (int i=0; i<operations; i++){
            
            String op = s.next().trim();
            
            if (op.charAt(0) == 'd'){
                sum += s.nextLong();
            } else {
                System.out.println(sum);
            }
        
        }

    }

}

File: OnlineJudge/P12403.c
#include <stdio.h>

int main(){

    int operations;
    scanf("%d", &operations);

    long long int sum = 0;
    long long int value;
    char word[15];

    while (operations--){

        scanf("%s", word);

        if (word[0] == 'd'){
            scanf("%lld", &value);
            sum += value;
        } else {
            printf("%lld\n", sum);
        }

    }

    return 0;
}